chart/charts/crds/templates/_helpers.tpl

```diff
@@ -0,0 +1,19 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{/*
+    This returns a "1" if the CRD is absent in the cluster
+    Usage:
+      {{- if (include "crdIsAbsent" (list <crd-name>)) -}}
+      # CRD Yaml
+      {{- end -}}
+*/}}
+{{- define "crdIsAbsent" -}}
+    {{- $crdName := index . 0 -}}
+    {{- $crd := lookup "apiextensions.k8s.io/v1" "CustomResourceDefinition" "" $crdName -}}
```

Comment:
what happens when there's a v2?

Reply:
What if we need to keep a mix of v1 and v2? Example let's say csi with v2 and jaeger with v1?

Reply:
Two tpl functions, maybe? crdIsAbsentv1 crdIsAbsentv2

Reply:
Yeah guess so. We can't do that dynamically by parsing the CR yaml?

Reply:
I've been poking around, and I could find a way to do that @tiagolobocastro.

Reply:
Do you think if the template function also took the apiVersion as input, it would be better?

Reply:
If you could find a way around it and parse yaml then I don't think we need to barcode the API Version on the template

Reply:
I'm sorry, I meant to write 'I couldn't find a way', I made a typo.